src/views/settings/Applet/AppletHost/AppletHostDetail/Applets.vue

```diff
@@ -171,7 +173,3 @@ export default {
   }
 }
 </script>
-
-<style scoped>
-
-</style>
```

Comment:
There is insufficient information provided to analyze the given code snippet thoroughly. Please specify the relevant sections or parts of the code you would like me to review carefully.